src/models/service/transactions/summary.rs

```diff
@@ -21,6 +22,14 @@ pub struct ExecutionInfo {
     pub missing_signers: Option<Vec<String>>,
 }
 
+#[derive(Serialize, Debug, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct SafeAppInfo {
```

Comment:
maybe have this in the `mod.rs`?

Reply:
I think I will move it to the `info.rs` with the rest of the stuff.